.github/workflows/zarr-dev.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Build with Zarr dev
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: ${{ matrix.platform }} ${{ matrix.python-version }}
+    runs-on: ${{ matrix.platform }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.7]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup miniconda
+        uses: conda-incubator/setup-miniconda@v1
```

Comment:
@Carreau : I'd suggest rolling back your codec change and bumping this to `v2`

Reply:
```suggestion
        uses: conda-incubator/setup-miniconda@v2
```

Reply:
do you think that's just a v2 issue ? I'm trying to understand why filter-id fails; and it's no clear to me.

Reply:
The v2 issue is just because github broke what miniconda was doing. I assume the codec/filter-id comes from the difference between n5 and zarr, but I haven't looked into it more closely. Constantin is the expert there, but
```
  File "numcodecs/blosc.pyx", line 546, in numcodecs.blosc.Blosc.__init__
TypeError: __init__() got an unexpected keyword argument 'codec'
```
suggests reverting.

Reply:
> The v2 issue is just because github broke what miniconda was doing

yeah, the ::set-path; but I don't believe this is yet fixed in v2. On other repositories I had to allow unsecure for those actions. 

I'll revert the doced and try to understand where the actual problem is.

Reply:
Re `::set-path:`, yes I also think this isn't fixed yet. I am fine with setting allow unsecure, I also did this in other repos to use `setup-miniconda`.